src/OpenTelemetry/CHANGELOG.md

```diff
@@ -38,6 +38,11 @@
   [IMetricsListener](https://learn.microsoft.com/dotNet/api/microsoft.extensions.diagnostics.metrics.imetricslistener).
   ([#5265](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5265))
 
+* **Experimental (pre-release builds only):** Obsoleted
```

Comment:
could we still call out that the `MeterProvider` still has a 2000 limit enforced for all metrics by default? We are only obsoleting the ability to change the MeterProvider level setting, in favor of view level setting.

Reply:
Updated